Hivemind/Assets/Scenes/Map.unity

```diff
@@ -106,15 +106,15 @@ NavMeshSettings:
   m_BuildSettings:
     serializedVersion: 2
     agentTypeID: 0
-    agentRadius: 0.03
```

Comment:
Why was this changed? Map no longer works because of this

Reply:
Good question. I don't remember editing that specifically

Reply:
Please revert this then, if this were merged develop would no longer work